lib/rubocop/cop/rspec/implicit_subject.rb

```diff
@@ -42,21 +42,36 @@ module RSpec
       #   # good
       #   it { expect(subject).to be_truthy }
       #
+      # @example `EnforcedStyle: require_implicit`
+      #   # bad
+      #   it { expect(subject).to be_truthy }
```

Comment:
As far as I understand, it would also apply to multi-line, multi-statement examples?